app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt

```diff
@@ -216,7 +217,8 @@ class FileInfoActionsView @JvmOverloads constructor(
             onItemClickListener.addFavoritesClicked()
         }
         leaveShare.setOnClickListener { onItemClickListener.leaveShare() }
-        // Use OnClickListener instead of OnCheckedChangeListener because the later is unnecessarily called on every
+        cancelExternalImport.setOnClickListener { onItemClickListener.cancelExternalImportClicked() }
+        // Use OnClickListener instead of OnCheckedChangeListener because the latter is unnecessarily called on every
         // refreshBottomSheetUI calls
```

Comment:
```suggestion
        // Use OnClickListener instead of OnCheckedChangeListener because the
        // latter is unnecessarily called on every refreshBottomSheetUI calls
```

Reply:
It would be a lack of consistance with the other commentaries